magic-entrypoint.py

```diff
@@ -22,19 +29,19 @@
     bind {listen}
     default_backend talk_{index}
 """
-config = """
+config = f"""
 global
     log stdout format raw daemon
 
 defaults
     log global
     mode tcp
-    timeout client 5s
-    timeout client-fin 5s
-    timeout connect 5s
-    timeout server 5s
-    timeout server-fin 5s
-    timeout tunnel 5s
+    timeout client {TIMEOUT_CLIENT}
+    timeout client-fin {TIMEOUT_CLIENT_FIN}
+    timeout connect {TIMEOUT_CONNECT}
+    timeout server {TIMEOUT_SERVER}
+    timeout server-fin {TIMEOUT_SERVER_FIN}
+    timeout tunnel {TIMEOUT_TUNNEL}
```

Comment:
Use `env(TIMEOUT_TUNNEL)` and similars, which is [supported natively by HAProxy](http://cbonte.github.io/haproxy-dconv/1.9/configuration.html#7.3.2-env).